env/posix/ocf_env.h

```diff
@@ -3,6 +3,14 @@
  * SPDX-License-Identifier: BSD-3-Clause-Clear
  */
 
+/**
+ * @file
+ * @brief OCF environment functions and macros
+ * 
+ * This file contains definitions for many OCF functions
+ * and sets macros
```

Comment:
I suggest edit:
This file contains definitions for OCF POSIX environment functions and macros.

Reply:
Changed as You suggested.